gettsim/benefits/arbeitslosengeld.py

```diff
@@ -15,22 +15,20 @@ def ui(df_row, tb):
 
     eligible = check_eligibility_alg(df_row)
 
-    if eligible:
-        if df_row["child_num_tu"] == 0:
-            return alg_entgelt * tb["agsatz0"]
+    if eligible.all():
```

Comment:
Why `.all()`? ALG1 is an individual benefit and only individuals should run through the function.

Reply:
The function is called on the individual level. Therefore the Series contains only one element!

Reply:
so there is no need for `.all()`?

Reply:
There should not be, indeed. 

Also, as you go through these things, can we please directly call them:
```
person
tax_unit
household
```
instead of `df_row` and `df`, respectively? If you insist, it can also be `p`, `tu`, `hh`. But this way it is clear immediately which level a functions touches.

Reply:
Will adapt it and use `iloc[0]` instead of `.all()`.

Reply:
But why is it not a scalar?

Reply:
When i use apply and on a groupby object which has single row groups, the functions are called with a one row dataframe. If I call by label, I get a one elemented Series.

Reply:
Then we should rewrite `check_eligibility_alg` so that it returns a scalar.

Reply:
Okay. So I shifted the iloc call to `check_eligibility_alg`.